src/Protocol/DFRN.php

```diff
@@ -177,8 +178,9 @@ public static function feed($dfrn_id, $owner_nick, $last_update, $direction = 0,
 			$groups = Group::getIdsByContactId($contact['id']);
 
 			if (count($groups)) {
-				for ($x = 0; $x < count($groups); $x ++)
+				for ($x = 0; $x < count($groups); $x ++) {
 					$groups[$x] = '<' . intval($groups[$x]) . '>' ;
+				}
```

Comment:
Can you please add a new line after this closing brace like you did with the `if () { ... }` blocks?